sdk/state/close.go

```diff
@@ -98,8 +108,7 @@ func (c *Channel) ConfirmCoordinatedClose(cc CoordinatedClose) (CoordinatedClose
 		cc.closeSignatures = append(cc.closeSignatures, txCoordinatedClose.Signatures()...)
 	}
 
-	// TODO - merge instead of overwrite, similar to ConfirmProposal
-	c.coordinatedClose = cc
+	c.coordinatedClose = mergeCoordinatedCloseData(c.coordinatedClose, cc)
 	return cc, nil
```

Comment:
We should return the merged result so that we're returning the final and complete state that we have.

Reply:
https://github.com/stellar/experimental-payment-channels/pull/88/commits/ec2b6306997dc887ef74c30809a0a1c40d822b5a